pkg/scheduler/framework/runtime/framework.go

```diff
@@ -327,7 +327,7 @@ func getPluginArgsOrDefault(pluginConfig map[string]runtime.Object, name string)
 		return res, nil
 	}
 	// Use defaults from latest config API version.
-	gvk := v1beta1.SchemeGroupVersion.WithKind(name + "Args")
```

Comment:
So, related to what I said here: https://github.com/kubernetes/kubernetes/issues/94008#issuecomment-703680713

If a removed Plugin is used via v1beta1 API and doesn't have a PluginConfig for it, this function will return nil and Plugin instantiation would fail.

Then, here we should try v1beta2 first and if it fails, try v1beta1.

More discussion will follow in #94008

Reply:
Let me accommodate this logic to try both the v1beta1 and v1beta2

Reply:
lets first just add v1beta2 and then make updates. We didn't remove any plugins yet, so introducing more complexity to this PR is not necessary.